sys-devel/binutils/binutils-2.41.recipe

```diff
@@ -0,0 +1,184 @@
+SUMMARY="Assembler, linker and binary tools for target ${targetMachineTriple}"
+DESCRIPTION="The GNU Binutils are a collection of binary tools.
+
+The main ones are:
+- ld - the GNU linker.
+- as - the GNU assembler.
+
+But they also include:
+- addr2line - converts addresses into filenames and line numbers.
+- ar - a utility for creating, modifying and extracting of archives.
+- c++filt - filter to demangle encoded C++ symbols.
+- gold - a linker for ELF files.
+- nm - lists symbols from object files.
+- objcopy - copys and translates object files.
+- objdump - displays information from object files.
+- ranlib - generates an index to the contents of an archive.
+- readelf - displays information from any ELF format object file.
+- size - lists the section sizes of an object or archive file.
+- strings - lists printable strings from files.
+- strip - discards symbols."
+HOMEPAGE="https://www.gnu.org/software/binutils/"
+COPYRIGHT="1988-2023 Free Software Foundation, Inc."
+LICENSE="GNU GPL v3
+	GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/gnu/binutils/binutils-$portVersion.tar.xz"
+SOURCE_DIR="binutils-$portVersion"
+CHECKSUM_SHA256="ae9a5789e23459e59606e6714723f2d3ffc31c03174191ef0d015bdf06007450"
+SOURCE_FILENAME="binutils-$portVersion.tar.xz"
+
+ARCHITECTURES="?all"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
Any reason why only for secondary architecture on 32bit?

Reply:
Besides the fact that I doubt this will build with gcc2, there is also my guess that the ABI changed with GCC 4 and that I suspect that binutils also changed to make that possible.

Reply:
I think the question was more about the "?all" in ARCHITECTURES, which means this is not built for x86_64 (as well as ARM, RISC-V, etc).

Reply:
Copy and paste error, will correct in the upcoming version.